commands/consume/consume.go

```diff
@@ -245,11 +245,15 @@ func filterError(err error) error {
 func closeFile(file *os.File, highlight bool) {
 	err := file.Sync()
 	if err != nil {
-		msg := fmt.Sprintf("Failed to sync the file: %s", err)
-		fmt.Println(format.Red(msg, highlight))
+		logger := internal.NewLogger(1)
```

Comment:
If you want to use logger, please use `VerbosityLevel` enum instead of the hard coded value of `1`